beacon-chain/rpc/apimiddleware/custom_hooks.go

```diff
@@ -284,16 +284,12 @@ func serializeProducedV2Block(response interface{}) (bool, []byte, gateway.Error
 	if strings.EqualFold(respContainer.Version, strings.ToLower(ethpbv2.Version_PHASE0.String())) {
 		actualRespContainer = &phase0ProduceBlockResponseJson{
 			Version: respContainer.Version,
-			Data: &beaconBlockContainerJson{
-				Message: respContainer.Data.Phase0Block,
-			},
+			Data:    respContainer.Data.Phase0Block,
```

Comment:
Does this need a regression test? Im surprised there's 0 test change to reflect on

Reply:
I improved block serialization tests.